app/templates/public/speakers.hbs

```diff
@@ -4,11 +4,6 @@
       {{public/speaker-item speaker=speaker}}
     </div>
   {{/each}}
-  {{#infinity-loader infinityModel=model}}
-    <div class="center aligned five wide column">
-      <div class="ui loading very padded basic segment">
-      </div>
-    </div>
-    {{infintyModel.reachedInfinity}}
+  {{#infinity-loader infinityModel=model triggerOffset=40 eventDebounce=30}}
```

Comment:
Why did you remove the loader?

Reply:
I just removed the spinner, infinityLoader is still there. Removing the spinner reduced the waiting on api host.

Reply:
That doesn't make any sense. Please add it back